plonky2/examples/poseidon2_bn254_example.rs

```diff
@@ -56,9 +56,9 @@ fn main() -> Result<()> {
     // fs::write("fibonacci_common_k.json" , common_circuit_data_serialized)       .expect("Unable to write file");
```

Comment:
we need JSON serialization support for the BN254 stuff to be able to read this in the Haskell verifier (and also for debugging / understanding it's extremely helpful)

optionally native "plonky2 binary serialization" too (it's somewhere in `utils` as I remember)